components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -1036,6 +1036,7 @@ class GeckoEngineSession(
         defaultSettings?.testingModeEnabled?.let { geckoSession.settings.fullAccessibilityTree = it }
         defaultSettings?.userAgentString?.let { geckoSession.settings.userAgentOverride = it }
         defaultSettings?.suspendMediaWhenInactive?.let { geckoSession.settings.suspendMediaWhenInactive = it }
+        defaultSettings?.clearColor?.let { geckoSession.compositorController.clearColor = it }
```

Comment:
Let's also add a test for `GeckoEngineSession` similar to https://github.com/mozilla-mobile/android-components/blob/master/components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt#L1257-L1283